### ISSUE TITLE
There is an error in the AccessorNode and IndexNode documentation example

### ISSUE THREAD

**Issue by rich1e** [josdejong/mathjs#2996]
There is an error in the AccessorNode and IndexNode documentation example.

**Describe the bug**

```
Uncaught TypeError: Array containing Nodes expected for parameter "dimensions"
```

![image](https://github.com/josdejong/mathjs/assets/29914689/d92e51dd-0eb2-4402-bb11-9519eb75fd84)

[Error Page](https://mathjs.org/docs/expressions/expression_trees.html#accessornode)

```
Uncaught TypeError: math.AccessNode is not a constructor
```

![image](https://github.com/josdejong/mathjs/assets/29914689/ed4e3ebb-4405-4ed1-8a33-981c937ab492)

[Error Page](https://mathjs.org/docs/expressions/expression_trees.html#indexnode)

**To Reproduce**
It can be reproduced in the browser, As shown in the picture above.

**Comment by josdejong** [comment]
Thanks for reporting, fixed via #2996

**Comment by josdejong** [comment]
Fix has been published now in `v11.9.0`
